src/test/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2IT.java

```diff
@@ -1507,4 +1508,74 @@ private Map<String, String> getConfig() {
       return TestUtils.getConfForStreamingWithOAuth();
     }
   }
+
+  // note this test relies on testrole_kafka and testrole_kafka_1 roles being granted to test_kafka
+  // user
+  @Test
+  public void testStreamingIngest_multipleChannel_distinctClients() throws Exception {
```

Comment:
confirmed this test fails with the same error as jira when run on master, passes with this pr's changes